docs/docs/widgets/button.md

```diff
@@ -3,3 +3,35 @@ sidebar_position: 1
 ---
 
 # Button
+
+Button widget can be used for allowing the users to take actions with a single click.
+
+<img class="screenshot-full" src="/img/widgets/button/adding-button.gif" alt="ToolJet - Widget Reference - Button" height="420"/>
+
+## Event: On click
+
+On click event is triggered when button is clicked and sets the action to run.
+
+
+:::info
+List of supported actions:
+* Show Alert
+* Run Query
+* Open Webpage
+* Go to app
+* Show Modal
+* Close Modal
+* Copy to clipboard
+:::
+
+<img class="screenshot-full" src="/img/widgets/button/button-actions-list.png" alt="ToolJet - Widget Reference - Button Action List" height="420" width="746"/>
+
+<img class="screenshot-full" src="/img/widgets/button/button-action.gif" alt="ToolJet - Widget Reference - Button Event" height="420" width="746"/>
+
+
+#### Properties
+
+| properties      | description |
+| ----------- | ----------- |
+| Loading state | Loading state on the button is used to show the loading status while action is being . Default value is `false` . |
```

Comment:
`while action is being` sentence is incomplete. 

Can we change this to: loading state can be used to show a spinner as the button content. Loading state is commonly used with `isLoading` property of the queries to show a loading status while a query is being run.

Reply:
yes this sounds better.